File: bot.js
const Discord = require("discord.js");
const fs = require("fs");
const config = require("./config.json");
const client = new Discord.Client();
client.commands = new Discord.Collection();

fs.readdir("./events/", (err, files) => {
	if(err) {
		console.error(err);
	}
	files.forEach(file => {
		let eventFunction = require(`./events/${file}`);
		let eventName = file.split(".")[0];
		client.on(eventName, (...args) => eventFunction.run(client, ...args));
	});
});

fs.readdir("./commands/", (err, files) => {
	if(err) return console.error(err);

	let jsfile = files.filter(f => f.split(".").pop() === "js");
	if(jsfile.length <= 0) {
		console.log("No commands found!");
		return;
	}

	jsfile.forEach((f, i) => {
		let props = require(`./commands/${f}`);
		client.commands.set(props.help.name, props);
		console.log(`${f} was loaded!`);
	});

});

client.on("message", message => {
	if(message.author.bot) return;
	if(!message.content.startsWith(config.prefix)) return;
	let prefix = (config.prefix);
	let messageArray = message.content.split(/ +/g);
	let cmd = messageArray[0];
	let args = messageArray.slice(1);

	let commandFile = client.commands.get(cmd.slice(prefix.length));
	if(commandFile) return commandFile.run(client, message, args);

});

client.login(process.env.bot_key);

File: bot.py
import discord
import asyncio

client = discord.Client()

@client.event
async def on_message(message):
  if messsage.content.startswith('!ping'):
    await client.send_message(message.channel, 'Your expecting me to say **Pong!**, right?')
    
@client.event
async def on_message(message):
  if message.content.startswith('!help'):
    await client.send_messsage(message.author, 'Commands are: **!help**, **!ping**, and **!ding**')
                              
@client.event
async def on_message(message):
  if message.content.startswith('!ding'):
    await client.send_message(message.author, 'Dong!')
      
client.run('MzY1NjQ5OTE4MTQyMDU0NDEw.DLiH3g.yrJsXqNQMeP7g6kFGN_DFo-zlts')
